### PR TITLE
Issue #465

### LINKED ISSUE #465: Add new labels as filters on "Explorer tasks"
Add extra labels on Explorer tasks, to show issues with bounties and contributions
(issues without bounties).

We need to highlight that new labels add new filters rather than select just one option. So we can have as example issues with `In progress` filter with `Contribution`.

### Why
We want to filter with extra filters, `$` for issues with bounties and `Contributions` to list issues without bounties. 

### Level
Easy

### Usefull links
  - https://github.com/reduxjs/reselect

### Screenshot attached
task-explore-labels

### How to solve this issue
 - Add new selectors and take care of the redux state to efficiently filter issues

### How to reproduce the user scenario

 - Sign in a new user
 - Import an issue
 - Check filters

### PR DESCRIPTION
**Pull request by VictorAlessander**
## Description

> Development of new labels to filter tasks in task list

## Changes

- Task List Filter
- Task List Table

## Issue

> Closes #465 

## Impacted Area

> Task list explorer

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Go to explore tasks
- Select a '$' or 'Contribution' filter label;
- Verify if the tasks filtered in task list are correct based on filter label selected;

### CODE REVIEW

**Comment by alexanmtz** [comment]
Hey @VictorAlessander, thanks for sending this PR in a proper time.

We need to review this solution, cause maybe the requirements weren't clear, but I meant: *So we can have as example issues with In progress filter with Contribution.*

There's a separator between the filter status on the layout: `|`

So we have one block of status (open, in progress, finished) and other block for these types ($ and contribution)

These filters are separated, so they are independent, you can select `in_progress` together with `contribution`, for example, to retrieve tasks for contribution with the status `in_progress`.

**Comment by VictorAlessander** [comment]
@alexanmtz Now, it's just missing the pipe to divide the main filters and additional filters and the bug fix that we already discussed too.

Also, if you allow me, how did you insert the pipe (divider)?
I tried with many approaches to use the Divider component that Material-Ui providers, but with no success.

**Comment by alexanmtz** [comment]
@VictorAlessander that's great 🎉 

And thanks for the update 🙏 

So, for the divider we can go in a simple way, using the character `|` and flexbox to align the items.

**Comment by VictorAlessander** [comment]
@alexanmtz Today wasn't a good day 😞 

I upgraded the packages of my environment (brew upgrade) and I spent a long time trying to understand why the project wasn't working, then, after try docker with no success (because of IP binding of containers) I realized that the Node was upgraded too. I was using a version 14.0.0 when the gitpay project needs of version 8.6.0.

Finally I builded successfully the project and started the implementations.

Sorry about that.

--

Well, I implemented the pipe (divider) successfully.

About the bug reported, I analyzed and understood that can take a while to fix. If you prefer to handle this bug in another issue, please, I would like to be assigned.

In short words, I have quite sure that the problem is in task-list (component) file.

**Comment by alexanmtz** [comment]
Hey @VictorAlessander, great job 🥇 

Can you create this issue with the steps to reproduce the bug?

Anyway, I will merge your changes and thanks for contributing 🙏

**Comment by VictorAlessander** [comment]
https://github.com/worknenjoy/gitpay/issues/644 created.
